components/shared/script/lib.rs

```diff
@@ -260,8 +260,9 @@ pub enum ScriptThreadMessage {
     SendImageKeysBatch(PipelineId, Vec<ImageKey>),
     /// Preferences were updated in the parent process.
     PreferencesUpdated(Vec<(String, PrefValue)>),
-    /// TODO
-    DisplayListDone(PipelineId),
+    /// Notify the `ScriptThread` that the Servo renderer is no longer waiting on canvas
+    /// images for the given `Pipeline`.
+    NoLongerWaitingOnCanvas(PipelineId),
```

Comment:
It might make sense to use more general name in the future, because I think we can move other stuff to this model too (svg, video?) although not normal images because they are async by design (onload).